Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
 groupsPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
+clientAuthorization=Autorisierung
+path=Pfad
+pairwiseSubAlgorithmSalt.label=Salz
```
IMHO "Salt" is a fixed term in hashing / encryption, and shouldn't be translated.

```suggestion
pairwiseSubAlgorithmSalt.label=Salt
```

ahus1 commented on js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:
```diff
@@ -206,3 +206,11 @@ samlRoleListScopeConsentText=Meine Rollen
 offlineAccessScopeConsentText=Offline Zugriff
 profileScopeConsentText=Benutzerprofil
 emailScopeConsentText=E-Mail Adresse
+organizations=Organisationen
+organizationDescription=Unternehmen ansehen, denen du beigetreten bist
+emptyUserOrganizations=Keine Unternehmen
+emptyUserOrganizationsInstructions=Du bist bisher noch keinem Unternehmen beigetreten.
+searchOrganization=Nach Unternehmen suchen
+organizationList=Liste an Unternehmen
```
IMHO we should translate it to "Organisation"/"Organisationen" consistently, and not switch to "Unternehmen" in some places.

robson90 replied:
I would go with "Organisation".

ahus1 commented on js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:
```diff
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
+createUser=Benutzer erstellen
+refreshTokens=Tokens aktualisieren
+kc.identity.authc.method=Authentifizierungsmethode
+clientProfiles=Kunden-Profile
```
"Client" might again be a untranslated term in the context of Keycloak, as it defines an application that is set up to work with Keycloak.

ahus1 reviewed (changes requested):
Thank you @Lxchnxr for these translations! See below for some comments. Please discuss here, and update the translations in Weblate. 

Also pinging @robson90 to provide his feedback on this as well, and possibly configure the untranslatable terms.

robson90 commented:
@ahus1 what about linking this page: https://hosted.weblate.org/browse/keycloak/glossary/
Inside the Readme: https://github.com/keycloak/keycloak/blob/main/docs/translation.md ?

We should discuss, if we want to add more to the glossary.

ahus1 commented:
@robson90 - I changed "Organisations" auf "Organisationen". Now it should be ready to merge.

@jonkoops - this PR shows nicely how this creates a commit per language. The downside: You don't see any more who change what in the Git history, still that information is available in Weblate.

weblate committed:
Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Updated translation for German

Language: de

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Robin <39960884+robson90@users.noreply.github.com>
Co-authored-by: Tobias Lechner <me@tobias-lechner.com>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Robin <39960884+robson90@users.noreply.github.com>
Signed-off-by: Tobias Lechner <me@tobias-lechner.com>

ahus1 commented:
@robson90 - 

> what about linking this page: https://hosted.weblate.org/projects/keycloak/glossary/de/
Inside the Readme: https://github.com/keycloak/keycloak/blob/main/docs/translation.md ?

Sounds like a good idea to me. It might be worth explaining a bit more what a Glossary is and what we use if for in this context.
